package me.chanjar.weixin.channel.bean.product.assistant;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import me.chanjar.weixin.channel.bean.base.WxChannelBaseResponse;

/**
 * 站内外商品属性映射响应。
 * @deprecated 请迁移至 {@link com.binarywang.wxjava.store.bean.product.assistant.ExternalProductMappingResponse}。
 */
@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
@Deprecated
public class ExternalProductMappingResponse extends WxChannelBaseResponse {

  private static final long serialVersionUID = -2267639791023044849L;

  /** 外部商品属性名。 */
  @JsonProperty("external_attribute_name")
  private String externalAttributeName;

  /** 外部商品属性值。 */
  @JsonProperty("external_attribute_value")
  private String externalAttributeValue;

  /** 内部商品属性名。 */
  @JsonProperty("internal_attribute_name")
  private String internalAttributeName;

  /** 内部商品属性值。 */
  @JsonProperty("internal_attribute_value")
  private List<String> internalAttributeValue;
}
